### DISCUSSION TITLE
Location Naming

### DISCUSSION THREAD

**Discussion by Phill93** [nautobot/nautobot#2233] [General]
Hi,

can someone give me a tip how to best create my location structure in 1.4.0?

I have several buildings that are designated with numbers (e.g. 101,102...) in these the rooms are again designated with numbers (e.g. 001, 101, 201).

I have now created the following location structure in my Nautobot instance: Building -> Floor -> Room (102 -> G -> 101)

If I now have the room 101 in several buildings, Nautobot lists me the 101 several times in the location field without differentiation.

<img width="905" alt="Bildschirmfoto 2022-08-17 um 17 40 31" src="https://user-images.githubusercontent.com/1454797/185182721-a7192cfa-f74b-4952-b1aa-ebdc9c3954f5.png">

Greetings

Daniel

Translated with www.DeepL.com/Translator (free version)

**Comment by bryanculver** [comment]
Hey @Phill93!

Looks like we could make some enhancements to this dropdown to include the Location Type in the display name in the dropdown. I will create an issue from this discussion.

Until then, you can name your Location to provide more context as to where it resides (`102 > G > 101`). Not ideal but it should give you a more helpful list at present.
